geometry_msgs/Vector3 pos_sp
geometry_msgs/Vector3 vel_ff
geometry_msgs/Vector3 acc_ff
float32 yaw_sp
